podpac/core/data/rasterio_source.py

```diff
@@ -140,10 +155,65 @@ def get_data(self, coordinates, coordinates_index):
         data.data.ravel()[:] = raster_data.ravel()
         return data
 
+    def get_data_overviews(self, coordinates, coordinates_index):
+        # Figure out how much coarser the request is than the actual data
+        reduction_factor = np.inf
+        for c in ["lat", "lon"]:
+            crd = coordinates[c]
+            if isinstance(crd, UniformCoordinates1d):
```

Comment:
You should use `if crd.is_uniform` here. The ArrayCoordinates1d have a `step` property.